contribs/drt/src/main/java/org/matsim/contrib/drt/run/DrtModeModule.java

```diff
@@ -20,18 +20,16 @@
 
 package org.matsim.contrib.drt.run;
 
+import com.google.inject.Key;
+import com.google.inject.Singleton;
+import com.google.inject.name.Names;
```

Comment:
General note (not limited to this PR): This PR contains lots of line changes due to different formatting standards, especially imports reordering/wildcarding. There are, for instance, no other changes in this file.

I hope this will get reduced after we switch to a formatting standard.

Reply:
Yes, I see. Guess it's from my auto-formatting in IntelliJ...